tasks/setup.yml

```diff
@@ -0,0 +1,7 @@
+---
+# Perform necessary setup
+- name: Install apt-transport-https to support HTTPS repository URLs
+  ansible.builtin.apt:
+    force_apt_get: yes
+    name: apt-transport-https
```

Comment:
Are more packages required here?  This disagrees with [this earlier code](https://github.com/cisagov/ansible-role-docker/blob/develop/tasks/setup_Debian.yml#L4-L11).

Reply:
That code is installing everything needs to add a PPA as well though.

Reply:
OK, got it.